dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -1583,16 +1601,13 @@ const CalpontSystemCatalog::TableColName CalpontSystemCatalog::dictColName(const
 
     return tableColName;
 }
-uint64_t CalpontSystemCatalog::nextAutoIncrValue ( TableName aTableName)
+uint64_t CalpontSystemCatalog::nextAutoIncrValue (TableName aTableName, int lower_case_table_names)
 {
-    transform( aTableName.schema.begin(), aTableName.schema.end(), aTableName.schema.begin(), to_lower() );
-    transform( aTableName.table.begin(), aTableName.table.end(), aTableName.table.begin(), to_lower() );
-
```

Comment:
No, I didn't miss it. This function calls tableInfo and passes the flag on. I thought this would be enough, but maybe it isn't. I all the if (lower_case_table_names) here since aTableName is used later in the function.

Reply:
Interestingly, this function is only called once with the flag set, and that is while processing system catalog, which is always lower case anyway. While the fix is made, it will have no functional change to the current behavior.

Reply:
I am seeing the call to this func with the 2nd argument from is_columnstore_tables_fill in . Is this what you are referring to?

Reply:
Yeah.